spec/Section 3 -- Type System.md

```diff
@@ -2164,3 +2164,128 @@ to the relevant IETF specification.
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
 ```
+
+## Schema Coordinates
+
+SchemaCoordinate :
+
+- Name
+- Name . Name
+- Name . Name ( Name : )
+- @ Name
+- @ Name ( Name : )
+
+:: A _schema coordinate_ is a human readable string that uniquely identifies a
+_schema element_ within a GraphQL Schema.
+
+:: A _schema element_ is a specific instance of a named type, field, input
+field, enum value, field argument, directive, or directive argument.
```

Comment:
Nit: "instance" has very much a "runtime" connotation to me. Could we maybe get rid of it?

```suggestion
:: A _schema element_ can be a named type, a field, an input
a field, an enum value, a field argument, a directive, or a directive argument.
```